app/models/miq_expression.rb

```diff
@@ -1874,46 +1865,6 @@ def self._quick_search?(e)
 
   private
 
-  class WhereExtractionVisitor < Arel::Visitors::PostgreSQL
```

Comment:
Yes :) It's still there but I moved it under another class now that MiqExpression is no longer using it